pettingzoo/test/api_test.py

```diff
@@ -386,11 +386,12 @@ def play_test(env, observation_0, num_cycles):
             assert env.observation_space(agent).dtype == prev_observe.dtype
         assert env.observation_space(agent).contains(
             prev_observe
-        ), "Out of bounds observation: " + str(prev_observe)
+        ) or env.observation_space(agent).contains(prev_observe[()]), "Out of bounds observation: " + str(prev_observe)
 
+        # Why the same assertion again???
         assert env.observation_space(agent).contains(
             prev_observe
-        ), "Agent's observation is outside of it's observation space"
```

Comment:
I think just removing this whole chunk is suffice

Reply:
I have seen that np.array(obj), meaning that the ndim is 0, or how I have accessed the item with [()], the "contains" fails. Then by checking the item itself, it works. So ndim shape np.array is not in the space, the object itself is.

Reply:
Seems to be my bug. Closing. If issue returns, I'll open a new PR.